mypy.ini

```diff
@@ -13,11 +13,45 @@ plugins = trio_typing.plugin
 ignore_errors = False
 disallow_untyped_defs=True
 disallow_any_unimported=True
-# disallow_any_expr=True
 disallow_any_decorated=True
 disallow_any_explicit=True
 disallow_any_generics=True
 disallow_subclassing_any=True
 
 [mypy-parsec.core.fs.workspacefs.versioning_helpers]
 disallow_untyped_defs=False
+
+
+[mypy-parsec.api.protocol.*]
```

Comment:
No common configuration for `parsec.api`?

Reply:
fixed